tests/test_switch.py

```diff
@@ -7,23 +7,33 @@
 
 def test_write_activator_switches(tmpdir):
     prefix = tmpdir / "prefix"
-    release = "2020.01.01-py27-rhel6"
-    expected_release = "2020.01.01-py27"
+    release = "2020.01.01-py38-rhel6"
+    expected_release = "2020.01.01-py38"
```

Comment:
What's the reason for this update? Did we have py38 on rhel6 even?

Reply:
I guess this is okay as is. I guess i'm just a bit put off by the combination in the version number. If the test only verifies that the suffixes are picked correctly, we could just as well update the rest of the version string to something more recent to avoid confusion?